### ISSUE TITLE
Even the example does not work

### ISSUE THREAD

**Issue by Teeed** [skorokithakis/sysaidmin#7]
Hi,

It looks like something is not working as it should be. Tried version from GIT and from pip (using pipx), result is always the same. Even the example from README does not seem to work.

```
$ sysaidmin "Whenever I try to SSH to localhost on port 22, it doesn't work. Can you fix it?"

==============================
It seems you are trying to display the result of a previously run command, but you have not provided the actual command or its output. Could you please provide more context or specify the command you executed so I can assist you with the results?
==============================

Your response: I was trying to run ssh localhost

==============================
It appears you're asking for the output of a command, but you haven't specified which command you would like me to run. If you provide me with a specific command to execute, I can help you obtain the output using the tools provided.
==============================

Your response: 
```

**Comment by aivanise** [comment]
I was just about to open an identical ticket, ubuntu 22.04 (python 3.10.12), using pipx

**Comment by skorokithakis** [comment]
This has been fixed and released as 0.1.2, thank you all!
